### PR TITLE
added license and miscellaneous card on modify product key screen and…

### PR DESCRIPTION
**Pull request by Suyashi123**
… refactored code

### CODE REVIEW

File: src/Components/CustomCard/CustomCardHeader.jsx [hunk 1]
```diff
@@ -102,108 +130,32 @@ const CustomCardHeader = ({ title, ...props }) => {
   };
 
   return (
-    <Box>
+    <Box mb={1.5} sx={{ height: 30 }}>
```
Comment (pinak-globallogic) [inline src/Components/CustomCard/CustomCardHeader.jsx:133]: remove mb and height

File: src/Pages/License/Details/Cards/MiscellaneousCard.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import Grid from "@mui/material/Grid";
+import CustomCard from "Components/CustomCard/CustomCard";
+import CustomCardContent from "Components/CustomCard/CustomCardContent";
+import CustomCardHeader from "Components/CustomCard/CustomCardHeader";
+//import CustomCardRow from "Components/CustomCard/CustomCardRow";
```
Comment (pinak-globallogic) [inline src/Pages/License/Details/Cards/MiscellaneousCard.jsx:5]: remove

File: src/Pages/License/Details/Cards/MiscellaneousCard.jsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+const MiscellaneousCard = () => {
+  const cardState = useSelector((state) => state.customCard.miscellaneousCard);
+  return (
+    <CustomCard>
+      <CustomCardContent>
+        <CustomCardHeader title={{id: cardState.id, name: "Miscellaneous", expanded: cardState.expanded}} />
+        <Collapse in={cardState.expanded} timeout="auto" unmountOnExit>
+        <Divider sx={{ borderBottomWidth: 2, mb:1 }} />
+        <Grid item container xs direction="column">
+            {/* <CustomCardRow title="License Server ID" value={data.name} mb={0} /> */}
```
Comment (pinak-globallogic) [inline src/Pages/License/Details/Cards/MiscellaneousCard.jsx:19]: remove

**Review by pinak-globallogic — CHANGES_REQUESTED** [review]

File: src/Components/CustomCard/CustomCardHeader.jsx (hunk 1 above)
Comment (Suyashi123) [reply src/Components/CustomCard/CustomCardHeader.jsx]: done

File: src/Pages/License/Details/Cards/MiscellaneousCard.jsx (hunk 3 above)
Comment (Suyashi123) [reply src/Pages/License/Details/Cards/MiscellaneousCard.jsx]: done

**Review by pinak-globallogic — APPROVED** [review]

**Pull request merged**